### PR TITLE
[bitnami/tomcat] Add tests and publish using VIB

### PR DESCRIPTION
**Pull request by joancafom**
**Description of the change**
The main objective of this PR is to publish our Bitnami Tomcat Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

**Benefits**

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

**Possible drawbacks**
Automated tests could show to be potentially flaky. 

**Test results**

Tested in https://github.com/joancafom/charts/pull/65

### Checklist

- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in the agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by recena — APPROVED** [review]

File: .vib/tomcat/cypress/cypress/integration/tomcat_spec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows to deploy a new WAR app', () => {
+  cy.visitAuth(`/manager`);
+  cy.contains('Web Application Manager');
+  cy.contains('form', 'Select WAR file').within(() => {
```
Comment (FraPazGal) [inline .vib/tomcat/cypress/cypress/integration/tomcat_spec.js:7]: Looking into simplifying the logic here, is this `contains` necessary? Could we make it work leaving only the `cy.get('[type="file"]')` below? Usually, there are no multiple elements with that selector.

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR! I left a couple of minor comments for this PR, please check them whenever possible @joancafom

File: .vib/tomcat/cypress/cypress/integration/tomcat_spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows to deploy a new WAR app', () => {
+  cy.visitAuth(`/manager`);
+  cy.contains('Web Application Manager');
```
[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/tomcat/cypress/cypress/integration/tomcat_spec.js:6]: Okay, fair enough!

File: .vib/tomcat/cypress/cypress/integration/tomcat_spec.js (hunk 1 above)
Comment (joancafom) [reply .vib/tomcat/cypress/cypress/integration/tomcat_spec.js:7]: It is true that there is only a single `[type="file"]`, but there are more than one `Deploy` buttons as you can see: 

deploy_form

The `form` selector is meant for that specific `Deploy` button, and not the `[type="file"]` field 😁

**Review by agomezmoron — DISMISSED** [review]
LGTM

**Review by agomezmoron — DISMISSED** [review]

**Review by agomezmoron — APPROVED** [review]

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**